sympy/stats/error_prop.py

```diff
@@ -0,0 +1,62 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
+
+
+def variance_prop(expr, consts=()):
+    """Symbolically propagates variance (sigma^2) for expressions.
+
+    Parameters
+    ==========
+    expr : Expr
+        A sympy expression to compute the variance for.
+    consts : sequence of Symbols, optional
+        Represents symbols that are known constants in the expr,
+        and thus have zero variance.
+
+    Returns
+    =======
+    var_expr : Expr
+        An expression for the total variance of the expr.
+        The variance for the original symbols (e.g. x) have a 'var_'
+        prepended to their name ('var_x').
+
+    Examples
+    ========
+
+    >>> from sympy import symbols, exp
+    >>> from sympy.stats.error_prop import variance_prop
+    >>> x, y = symbols('x y')
+
+    >>> variance_prop(x + y)
+    var_x + var_y
+
+    >>> variance_prop(x * y)
+    var_x*y**2 + var_y*x**2
+
+    >>> variance_prop(exp(2*x))
+    4*var_x*exp(4*x)
+
+    """
+    args = expr.args
+    if len(args) == 0:
+        if isinstance(expr, Symbol) and expr not in consts:
```

Comment:
Is RandomSymbol supposed to be a user-facing class? It isn't in `sympy/stats/__init_.py` and doesn't have a doctest in its docstring. 

Another question, would forcing the use of RandomSymbol put a burden on the user to create distinct probability spaces (with mean and variance) for each symbol? It seems RandomSymbol works like `RandomSymbol(pspace, symbol)`. But in this function, we already assume each symbol has its own independent probability space (if I understand correctly). Starting with `x` and then creating `variance(x)` is more convenient here than starting with `mu_x` and `sigma_x` and creating x from them.